### PR TITLE
Feat: Roll functionality

### PR DESCRIPTION
**Pull request by AdvanceRatio**

### CODE REVIEW

File: cogs/roll_cog.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import discord
+from discord.ext import commands
+import random
+
+
+class Example(commands.Cog):
```
Comment (MFry) [inline cogs/roll_cog.py]: This should be renamed to what the cog actually does

**Review by MFry — CHANGES_REQUESTED** [review]
Sweet stuff

File: cogs/dice_roller.py [hunk 2]
```diff
@@ -3,38 +3,30 @@
...
 
-class Example(commands.Cog):
+class DiceRoller(commands.Cog):
     def __init__(self, client):
         self.client = client
 
     @commands.Cog.listener()
     async def on_ready(self):
-        print('Example cog is loaded')
+        print('Dice rolling cog is loaded')
```
Comment (MFry) [inline cogs/dice_roller.py]: Current pattern is `{cog_name} *cog* is loaded`

File: cogs/dice_roller.py [hunk 3]
```diff
@@ -3,38 +3,30 @@
...
         dice_number = int(dice_list[0])
-
         dice_type = int(dice_list[1])
-
         if dice_number == 1:
-
             dice_total = random.randint(1, dice_type)
-
             await ctx.send(
                 f'*{ctx.message.author} rolls {dice} for {dice_total}*'
```
Comment (MFry) [inline cogs/dice_roller.py:38]: Demo the dice rolling please.

**Review by MFry — CHANGES_REQUESTED** [review]

File: cogs/dice_roller.py (hunk 3 above)
Comment (AdvanceRatio) [reply cogs/dice_roller.py:38]: image

File: cogs/dice_roller.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+class DiceRoller(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+
+    @commands.Cog.listener()
+    async def on_ready(self):
+        print('Dice Roller *cog* is loaded')
+
+    @commands.command()
+    async def roll(self, ctx, dice: str):
```
Comment (MFry) [inline cogs/dice_roller.py:20]: Please add docs and how to use the commands ([good reference](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html)
 """
        
        :param ctx: 
        :param dice
        :type dice: str
        :return: 
        """

**Review by MFry — CHANGES_REQUESTED** [review]
Maybe we can add helper functions

**Review by MFry — APPROVED** [review]
👍 🚀

**Pull request merged**